.github/actions/build-android/action.yml

```diff
@@ -0,0 +1,104 @@
+name: Android Build
+description: Installs required dependencies and then builds the Android side of the SDK 
+
+inputs:
+  node-version:
+    description: The version of Node.js to use
+    required: true
+  java-version:
+    description: The version of Java to use
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Dependencies including 32-bit support
+      run: |
+        sudo dpkg --add-architecture i386
+        sudo apt-get update -y -qq
+        sudo apt-get install -y -qq gperf libatomic1:i386 libc6:i386 libncurses5:i386 libstdc++6:i386
+      shell: bash
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ inputs.node-version }}
+        cache: 'npm'
+
+    - name: Setup Java
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'adopt'
+        java-version: ${{ inputs.java-version }}
+
+    - name: Install dependencies
+      run: npm ci
```

Comment:
I don't think so, my understanding of the setup-node cache is that it's for the global modules directory not a projects node_modules. So that key tells us if setup-node restored the cached global node_modules directory not a local node_modules for a project

[ref](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-dependencies)